Documentation/k8s-pre-reqs.md

```diff
@@ -22,6 +22,30 @@ Creating the Rook operator requires privileges for setting up RBAC. To launch th
 -subj "/CN=admin/O=system:masters"
 ```
 
+## Flexvolume Configured
+
+Rook uses [Flexvolume](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md) to integrate with Kubernetes for performing storage operations. In some operating systems where Kubernetes is deployed, the [default Flexvolume plugin directory](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md#prerequisites) (the directory where flexvolume drivers are installed) is **read-only**.
+
+This is the case for Kubernetes deployments on CoreOS and Rancher. In these environments, the flexvolume plugin directory needs to be specified to be different than the default. This is done by passing the `--volume-plugin-dir` flag to Kubelet and restarting it. This should be done on all Kubelets in all nodes.
+
+:information_source: **CoreOS**
```

Comment:
i personally think these information source icons get rendered hideously and look really bad.  i would consider using just level 3 or 4 header instead.